### PR TITLE
For Youtube data-videoid, require that the value conforms to [^=/\?:]+

### PR DESCRIPTION
**Pull request by powdercloud**
Specifically this disallows YouTube URLs, which aren't supported.

### CODE REVIEW

**Comment by Gregable** [comment]
lgtm

**Comment by aghassemi** [comment]
:+1: and thanks for the change. Will definitely reduce confusion. Hopefully it is not invalidating too many pages but either way, their videos would not have been working, so this leads to better user experience for validated AMP pages.

File: extensions/amp-youtube/0.1/validator-amp-youtube.protoascii [hunk 1]
```diff
@@ -55,12 +55,14 @@ tags: {  # <amp-youtube>
   attrs: {
     name: "data-videoid"
     mandatory_oneof: "['data-videoid', 'video-id' (deprecated)]"
+    value_regex: "[^=/\?:]+"
```
Comment (jridgewell) [inline extensions/amp-youtube/0.1/validator-amp-youtube.protoascii]: I don't think the `?` needs escaping.

Comment (powdercloud) [inline extensions/amp-youtube/0.1/validator-amp-youtube.protoascii]: Thank you, fixed.

**Pull request merged**